.github/workflows/continuous-integration.yml

```diff
@@ -44,6 +44,7 @@ jobs:
       SLACK_WEBHOOK_PONTOON: ${{ secrets.SLACK_WEBHOOK_PONTOON }}
       SSL_REDIRECT: False
       USE_S3: False
+      DONATE_NEWSLETTER_SUBSCRIBE_METHOD: BASKET
```

Comment:
Updated the env var name to be clearer for future tasks involving SQS. Thanks to @benhohner for this suggestion!